metasv/main.py

```diff
@@ -102,6 +102,11 @@ def run_metasv(args):
         for native_file in nativename:
             for record in svReader(native_file, svs_to_report=args.svs_to_report):
                 interval = record.to_sv_interval()
+                
+                BD_min_inv_len = args.mean_read_length+4*args.isize_sd
+                if toolname=="BreakDancer" and interval.sv_type == "INV" and  abs(interval.length)< BD_min_inv_len:
+                    #Filter BreakDancer (~200,000) artifact INVs with size < readlength+4*isize_sd
```

Comment:
Don't need the 200,000 number as it is from our own testing of one dataset. This number can be way different for other  datasets. Please remove this number.